Makefile

```
@@ -1,24 +1,26 @@
+MAKE := make -i
+DOCKER_COMPOSE := ~/Downloads/docker-compose-darwin-x86_64
```

up: juha
down: jaeywon
```suggestion
DOCKER_COMPOSE := ~/Downloads/docker-compose-darwin-x86_64
//DOCKER_COMPOSE := docker compose
```